.travis.yml

```diff
@@ -20,6 +20,24 @@ matrix:
       python: 3.5
     - os: linux
       python: 2.7
+    - os: linux
+      if: type = cron
+      # The pytest major version is constrained to 3 because running the
+      # distributed test suite with pytest 4 fails for now
```

Comment:
Please insert a comment to add that this is an integration test to check the impact of cloudpickle on the downstream project dask-distributed.